bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -1628,60 +1628,195 @@ pimcore.helpers.searchAndMove = function (parentId, callback, type) {
 };
 
 
-pimcore.helpers.sendTestEmail = function () {
+pimcore.helpers.sendTestEmail = function (from = null, to = null, subject = null, emailType = null, documentPath = null, content = null) {
+
+
+    var emailContentTextField = new Ext.form.TextArea({
+        name: "content",
+        fieldLabel: t("content"),
+        height: 300,
+    });
+    emailContentTextField.hide();
+
+    var documentTextField = new Ext.form.TextField({
+        name: 'documentPath',
+        flex: 1,
+        editable: false
+    });
+    var searchDocumentButton = new Ext.Button({
+        name: 'searchDocument',
+        fieldLabel: t('document'),
+        iconCls: 'pimcore_icon_open',
+        handler: function() {
+            pimcore.helpers.itemselector(false, function(e) {
+                documentTextField.setValue(e.fullpath);
+            }, {
+                type: ["document"],
+                subtype: {
+                    document: ["email"]
+                }
+            });
+        }
+    });
+
+    var documentComponent = Ext.create('Ext.form.FieldContainer', {
+        fieldLabel: t('document'),
+        layout: 'hbox',
+        items: [
+            documentTextField,
+            searchDocumentButton
+        ],
+        componentCls: "object_field",
+        border: false,
+        style: {
+            padding: 0
+        }
+    });
+    documentComponent.hide();
+
+
+    var emailTypeDropdown = new Ext.form.ComboBox({
+        name: 'emailType',
+        width: 300,
+        store: [
+            ['document', t('document')],
+            ['html', t('html')],
+            ['text', t('text')]
+        ],
+        fieldLabel: t('type'),
+        listeners: {
+            select: function(t) {
+                if(t.value == 'text' || t.value == 'html') {
+                    emailContentTextField.show();
+                } else {
+                    emailContentTextField.hide();
+                }
+
+                if(t.value == 'document') {
+                    documentComponent.show();
+                    paramGrid.show();
+                } else {
+                    documentComponent.hide();
+                    paramGrid.hide();
+                }
+            }
+        }
+    });
+
+    var fromTextField = new Ext.form.TextField({
+        name: "from",
+        fieldLabel: t("from"),
+    });
+
+    var toTextField = new Ext.form.TextField({
+        name: "to",
+        fieldLabel: t("to"),
+    });
+
+    var subjectTextField = new Ext.form.TextField({
+        name: "subject",
+        fieldLabel: t("subject"),
+    });
+
+    var paramsStore = new Ext.data.ArrayStore({
+        fields: [
+            {name: 'key', type: 'string'},
+            {name: 'value', type: 'string'}
+        ]
+    });
+
+    var paramGrid = Ext.create('Ext.grid.Panel', {
+        store: paramsStore,
+        columns: [
+            {
+                text: 'key',
```

Comment:
we should use a a translation key here: `t("key")`

Reply:
done